packages/@aws-cdk/aws-apigatewayv2/lib/http/integrations/private-integration/application-loadbalancer-integration.ts

```diff
@@ -0,0 +1,25 @@
+import * as elbv2 from '@aws-cdk/aws-elasticloadbalancingv2';
+import { BasePrivateIntegrationProps, HttpProxyPrivateIntegration } from './http-private';
+
+/**
+ * Properties to initialize `ApplicationLoadBalancerPrivateIntegration`.
+ */
+export interface ApplicationLoadBalancerPrivateIntegrationProps extends BasePrivateIntegrationProps {
```

Comment:
As mentioned here - https://github.com/aws/aws-cdk/pull/10531#discussion_r500864507 - these will need to be spun off into a new module `aws-apigatewayv2-integrations`.

To keep the scope contained, implement only private integration the one that you need, but it's best to set up the correct pattern for those to follow.

Let me know how I can help with this.

Reply:
Done!